.rubocop.yml

```diff
@@ -0,0 +1,15 @@
+AllCops:
+  TargetRubyVersion: 2.4
+  Include:
+    - '**/Rakefile'
+    - '**/config.ru'
+  Exclude:
+    - db/schema.rb
+    - 'config/**/*'
+    - 'script/**/*'
+Documentation:
```

Comment:
I think this is one of the most ignored rules, only if the team agrees to document everything would make sense to enable... else a lot of issues on the rubocop warning list would appear... and people stop taking time to read them.